examples/sveltekit/src/routes/__layout.svelte

```diff
@@ -1,6 +1,21 @@
 <script>
   import Header from '$lib/header/Header.svelte';
+  import { webVitals } from '$lib/vitals';
+  import { browser } from '$app/env';
+  import { page } from '$app/stores';
   import '../app.css';
+
+  let analyticsId = import.meta.env.VERCEL_ANALYTICS_ID;
+
+  if (browser && analyticsId) {
+    page.subscribe(({ url, params }) =>
+      webVitals({
+        path: url.pathname,
+        params,
+        analyticsId
+      })
+    );
+  }
```

Comment:
It's _possible_ for this layout to get unmounted and remounted, even though it's the root layout, if a page somewhere uses a named layout that doesn't inherit from the root. In that case this subscription would keep getting re-added. We plan to add a universal entry point so that stuff like this doesn't need to be put in the root layout, but in the meantime this would work:

```suggestion
  $: if (browser && analyticsId) {
    webVitals({
      path: $page.url.pathname,
      params: $page.params,
      analyticsId
    })
  }
```